test/py/test_quiesce.py

```diff
@@ -195,36 +195,6 @@ def test_allowed_funcs_in_quiesced_dma_unregister_busy(mock_quiesce,
     mock_dma_unregister.assert_called_once_with(ctx, mock.ANY)
 
 
-@patch('libvfio_user.migr_trans_cb', side_effect=_side_effect)
-@patch('libvfio_user.quiesce_cb')
-def test_allowed_funcs_in_quiesed_migration(mock_quiesce,
-                                            mock_trans):
-
-    global ctx, sock
-    _map_dma_region(ctx, sock)
-    data = VFIO_DEVICE_STATE_V1_SAVING.to_bytes(c.sizeof(c.c_int), 'little')
```

Comment:
Shouldn't these be replaced with which migration states allow these functions in quiesced state?

Reply:
You're absolutely right, I also didn't implement this in the implementation branch but I've done that now (will push in a moment).